test/box-luatest/gh_7356_index_parts_methods_test.lua

```diff
@@ -0,0 +1,148 @@
+local t = require('luatest')
```

Comment:
I would add one `:alter()` case to verify that the methods are actually updated. Say, we can add a field into existing index.

Reply:
Done.